connectors/migrations/db/migration_64.sql

```diff
@@ -0,0 +1,2 @@
+-- Migration created on Apr 09, 2025
+ALTER TABLE "public"."salesforce_configurations" ADD COLUMN "usePersonalConnections" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
We might want true by default for new connections (default to the safest option)

Reply:
since it's behind a ff, i'd rather keep it to false ?